install/host_preparation.adoc

```diff
@@ -40,8 +40,12 @@ The `PATH` for the root user on each host must contain the following directories
 - *_/usr/bin_*
 - *_/usr/sbin_*
 
-These directories set by default in a new RHEL 7.x installation.
+These directories set by default in a new RHEL 7.x installation. Run the following command to verify your `PATH` settings:
 
+----
+echo $PATH
```

Comment:
\#

Make the addition a new paragraph, and I'd say "To verify your \`PATH\` settings, run the following command:"